pkg/nodes/node.go

```diff
@@ -22,6 +29,28 @@ type Builder struct {
 	errorMsg   string
 }
 
+// NewBuilder creates new instance of node.
+func NewBuilder(apiClient *clients.Settings, name string) *Builder {
```

Comment:
Do you have a use case for creating a node?

Reply:
no